website/docs/tools/analyzer.md

```diff
@@ -6,7 +6,11 @@ sidebar_position: 1
 
 The *analyzer* is a Software Composition Analysis (SCA) tool that determines the dependencies of software projects inside the specified version-controlled input directory (`-i`).
 It is the only mandatory tool to run from ORT as its output is the input for all other tools.
-It does so by querying the detected package managers; **no modifications** to your existing project source code, like applying build system plugins, are necessary for that to work.
+Analysis works by querying the detected package managers; **no modifications** to your existing project source code, like applying build system plugins, are necessary for that to work if the following preconditions are met:
+
+* All projects use one of the package managers listed below in a reasonably recent version, and they are configured according to common best practices.
+* All projects can be built in a single step out-of-the-box, without any custom confguration being set, like build system properties or environment variables.
```

Comment:
Is it worth to add that definition files use common / default file names?

Reply:
I would count that as a "common best practice".